simple/utils/astrometry.py

```diff
@@ -357,3 +357,181 @@ def ingest_proper_motions(
             updated_source_pm_data.pprint_all()
 
     return
+
+
+def ingest_radial_velocity(
+    db: Database,
+    *,
+    source: str = None,
+    rv: Union[Quantity, float] = None,
+    rv_err: Optional[Union[float, Quantity]] = None,
+    reference: str = None,
```

Comment:
I think the types here (and the docstring descriptions) are incorrect: these are not strings/floats/Quantities, but lists of string/floats/Quantities (or numpy arrays/astropy Tables or similar)

Reply:
Actually never mind, I think I confused `ingest_radial_velocity` with `ingest_radial_velocities`

Reply:
I find it interesting (and confusing) that this function takes a single value while the others in this module take lists of values- we should try to be consistent if possible.

Reply:
I still hold that all functions in utils should be built to be single entry ingests but with like a `np.vectorize` decorator